tests/baselines/reference/classStaticBlock22.errors.txt

```diff
@@ -1,94 +0,0 @@
-tests/cases/conformance/classes/classStaticBlock/classStaticBlock22.ts(4,9): error TS1359: Identifier expected. 'await' is a reserved word that cannot be used here.

```

Comment:
why have this and a bunch of other files been deleted?

Reply:
@gabritto Hi! They're baselines for tests that no longer exist --- they're removed by the Accept Baselines and Fix Lints workflow:
- https://github.com/microsoft/TypeScript/blob/6aaefe41fe03e4e67eda2f2c81df57adc08fd25b/.github/workflows/accept-baselines-fix-lints.yaml#L23
- https://github.com/microsoft/TypeScript/pull/45916